File: Swayy03/FizzBuzz.py
def fizzBuzz():
    for i in range(0, 100):
        if i%15 == 0:
            print("FizzBuzz")
        elif i%3 == 0:
            print("Fizz")
        elif i%5 == 0:
            print("Buzz")
        else:
            print(i)


fizzBuzz()

File: Swayy03/FizzBuzz.cpp
#include <iostream>

using namespace std;
void fizzBuzz();
int main(){
    fizzBuzz();

    return 0;
}

void fizzBuzz(){
    for(int i=0;i<=100;i++){
        if (i%15 == 0){
            cout<<"FizzBuzz"<<endl;
        }
        else if(i%3 == 0){
            cout<<"Fizz"<<endl;
        }
        else if(i%5 == 0){
            cout<<"Buzz"<<endl;
        }
        else{
            cout<<i<<endl;
        }
    }
}
